caragon4695/travel-tracker-final
# Cma/feature/trip class build

## Pull Request

<br>

### Description

* Is this a feature or a fix?

feature

* Where should the reviewer start?

Start at the `Destination` class and test files. Then move onto the `Trip` testing and class files.

* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?

They add functionality to the trip objects.

* What outside features or research did you use to implement this fix?

Moment.js

* How should this be tested?

Run npm test to make sure the tests are passing.

* Applicable screenshot(s): 

<br>

***

#### Please review considerations below:
